### PR TITLE
[refactor] Remove BASEOKxxx aliases for enum

### PR DESCRIPTION
**Pull request by ibuclaw**

### CODE REVIEW

File: src/dmd/aggregate.h [hunk 1]
```diff
@@ -44,7 +44,7 @@ enum Sizeok
 
 enum Baseok
 {
-    BASEOKnone,         // base classes not computed yet
```
Comment (wilzbach) [inline src/dmd/aggregate.h]: Accidental?

**Review by JinShil — APPROVED** [review]

File: src/dmd/aggregate.h (hunk 1 above)
Comment (ibuclaw) [reply src/dmd/aggregate.h]: Oops

**Comment by ibuclaw** [comment]
Reverted oops.

**Comment by wilzbach** [comment]
(rebased to restart DAutotest)

**Pull request merged**